subprocess/local_command.go

```diff
@@ -14,13 +14,13 @@ import (
 )
 
 type LocalCommand struct {
-	CmdString        string
-	WorkingDirectory string
-	Shell            string
-	Environment      []string
-	ScriptMode       bool
-	Stdout           io.Writer
-	Stderr           io.Writer
+	CmdString        string    `json:"command"`
+	WorkingDirectory string    `json:"directory"`
+	Shell            string    `json:"shell"`
+	Environment      []string  `json:"environment"`
+	ScriptMode       bool      `json:"script"`
+	Stdout           io.Writer `json:"-"`
```

Comment:
What's the difference between having this struct tag and leaving it blank?

Reply:
if it's public, and doesn't have a struct tag, then it is marshaled using the field name from the struct in json as is. this prevents it from being marshaled.